File: Java/710.java
import java.util.Scanner;

/**
 *
 * 710 - Plazas Disponibles en un Mundial - Golgotha's Melancholy - https://github.com/Golgotha-sMelancholy
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int casos = sc.nextInt();
        for (int i = 1; i <= casos; i++) {
            int equipos = sc.nextInt();
            int europa = sc.nextInt();
            int asia = sc.nextInt();
            int africa = sc.nextInt();
            int norteAmerica = sc.nextInt();
            int surAmerica = sc.nextInt();
            int oceania = sc.nextInt();
            System.out.println(equipos - europa - asia - africa - norteAmerica - surAmerica - oceania);
        }
    }
}

File: C++/710.cpp
#include <iostream>

using namespace std;

// 710 - Plazas Disponibles en el Mundial - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop = 0;
    cin >> loop;
    for(int i = 0; i<loop; i++) {
        int teams = 0;
        int europe = 0;
        int asia = 0;
        int africa = 0;
        int northAmerica = 0;
        int southAmerica = 0;
        int oceania = 0;
        cin >> teams;
        cin >> europe;
        cin >> asia;
        cin >> africa;
        cin >> northAmerica;
        cin >> southAmerica;
        cin >> oceania;
        cout << teams-europe-asia-africa-northAmerica-southAmerica-oceania << endl;

    }

}
